### PR TITLE
Fixed #137: bump skupper-router version to 2.0.0-SNAPSHOT

### LINKED ISSUE #137: Bump skupper-router version to 2.0.0

### PR DESCRIPTION
**Pull request by kgiusti**

### CODE REVIEW

File: docs/books/common/attributes.adoc [hunk 1]
```diff
@@ -44,7 +44,7 @@ under the License
 :RouterName: Dispatch Router
 :RouterSchemaDir: ../../build/doc/book
 :RouterConfigFile: /etc/qpid-dispatch/qdrouterd.conf
-:DispatchRouterVersion: 1.19.0-SNAPSHOT
+:DispatchRouterVersion: 2.0.0-SNAPSHOT
```
Comment (ganeshmurthy) [inline docs/books/common/attributes.adoc:47]: Should this be called SkupperRouterVersion ?

Comment (jiridanek) [reply docs/books/common/attributes.adoc:47]: I'm never sure what is and what is not in scope of each of these 'dispatch to skupper' transition PRs.

Comment (kgiusti) [reply docs/books/common/attributes.adoc:47]: @ganeshmurthy No - I simply ran the version update process I usually do for releasing qpid dispatch.   There's about a gazillion (give or take a few bazillion) uses of the term "dispatch" throughout the code.  This patch does nothing to change that.  All this patch does is change the version number.  And changes to documentation are way way way way way (did I say 'way') beyond the scope of this patch.

Comment (ganeshmurthy) [reply docs/books/common/attributes.adoc:47]: Perhaps https://github.com/skupperproject/skupper-router/pull/141/files could address this?

**Pull request merged**